src/all/hitomi/src/eu/kanade/tachiyomi/extension/all/hitomi/Hitomi.kt

```diff
@@ -229,7 +229,7 @@ class Hitomi(
         query: String,
         language: String = "all",
     ): MutableList<Int> {
-        query.replace("_", " ").let {
+        query.let { if (language != "all") query + " language:$nozomiLang" else query }.replace("_", " ").let {
```

Comment:
it won't work here, you need to add this to `hitomiSearch`. add `language:$nozomiLang` to the `terms` list

Reply:
well `getGalleryIDsForQuery` is used for individual query terms, so it can't handle two terms together

Reply:
Right. My fault.